contents/blog/startup-tigris.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Startup spotlight - Tigris Data"
+date: 2023-04-20
+author: ["charles-cook"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/transparency.jpeg
+featuredImageType: full
+category: Startups
+---
+
+This month we're chatting to Tigris co-founder & CEO Ovais Tariq. Prior to starting Tigris, Ovais spent several years at Uber where he was Head of Storage Infra.
+
+> Every month we're shining the spotlight on a different startup to show off the cool things they are building, as part of the PostHog for Startups program. Want to be in with a chance of getting featured? [Sign up here for $50k in PostHog credit and a bunch of other sweet perks!](/startups) 
+
+### What is Tigris and why is it cool?
+
+Tigris is a developer data platform that provides an integrated suite of data services that are needed by application developers to build data-driven applications. These services include a NoSQL database for storing structured and unstructured data, and an integrated search engine that allows users to do both full-text and vector searches. We've got more services coming soon, including event streaming and caching.
+
+We chose these services because most real-time applications that are built require a combination of these services. If you think of an e-commerce application, you need a way to store the product data, store the user data, store the orders-management data, a way to search everything etc. Usually you need a combination of different types of products to do this. 
```

Comment:
```suggestion
We chose these services because most real-time applications require a combination of them. If you think of an e-commerce application, you need a way to store the product data, store the user data, store the orders-management data, a way to search everything etc. Usually you need a combination of different types of products to do this. 
```